aws-observability-terraform/README.md

```diff
@@ -1,79 +1,41 @@
-# AWS Observability Terraform
-
-The AWS Observability terraform is used to deploy Apps and AWS sources for multiple accounts and Regions.
-
-For Multi account and regions use provider aliases as shown below
-
-```terraform
-# Region US-EAST-1, Account Production
-provider "aws" {
-  profile = "production"
-  region  = "us-east-1"
-  alias   = "production-us-east-1"
-}
-
-module "production-us-east-1" {
-  source    = "../source-module"
-  providers = { aws = aws.production-us-east-1 }
+## Requirements
 
-  aws_account_alias         = "production"
-  sumologic_organization_id = "0000000000123456"
-}
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13.0 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 3.42.0 |
+| <a name="requirement_random"></a> [random](#requirement\_random) | >= 3.1.0 |
+| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | >= 2.9.0 |
+| <a name="requirement_time"></a> [time](#requirement\_time) | >= 0.7.1 |
 
```

Comment:
Isn't REST API provider also needed by various modules?

Reply:
Not using restAPI provider anywhere in AWS Obs Soln.